Assets/VRTK/SDK/Daydream/SDK_DaydreamController.cs

```diff
@@ -0,0 +1,266 @@
+        {
+            var returnCollider = "ControllerColliders/Fallback";
+            return returnCollider;
+        }
+
+        public override string GetControllerElementPath(ControllerElements element, ControllerHand hand, bool fullPath = false)
+        {
+            //TODO: Use Gvr's tooltips or add an attach object ourselves
+            string dd = "Controller/ddcontroller/";
+            string pad = dd + "Tooltips/TouchPadOutside";
+            string app = dd + "Tooltips/AppButtonOutside";
+
+            switch (element)
+            {
+                case ControllerElements.AttachPoint:
+                    return pad; //TODO: attach point at tip of controller?
+                case ControllerElements.Touchpad:
+                    return pad;
+                case ControllerElements.ButtonOne:
+                    return app;
+                default:
+                    return dd;
+            }
+        }
+
+
+        public override uint GetControllerIndex(GameObject controller)
+        {
+            uint index = 0;
+
+            switch (controller.name)
+            {
+                case "Camera":
+                    index = 0;
+                    break;
+                case "RightController":
+                    index = 1;
+                    break;
+                case "LeftController": //Will be ignored; REMOVE OR RETURN 1 ?
+                    index = 2;
+                    break;
+            }
+            return index;
+        }
+
+        public override GameObject GetControllerByIndex(uint index, bool actual = false)
+        {
+            switch (index)
+            {
+                case 1:
+                    return controller;
+                default:
+                    return null;
+            }
+        }
+
+        public override Transform GetControllerOrigin(GameObject controller)
+        {
+            return controller.transform;
+        }
+
+
+        //public override Transform GenerateControllerPointerOrigin(GameObject parent)
+
+        //public override GameObject GetControllerLeftHand(bool actual = false)
+
+        public override GameObject GetControllerRightHand(bool actual = false)
+        {
+            controller = GetSDKManagerControllerRightHand(actual);
+            if ((controller == null) && actual)
+            {
+                controller = GameObject.Find("GvrControllerPointer/Controller");
+            }
+            if (controller != null)
+            {
+                prevPosition = controller.transform.position;
+                prevRotation = controller.transform.rotation.eulerAngles;
+            }
+            return controller;
+        }
+
+
+        //public override bool IsControllerLeftHand(GameObject controller)
+        //public override bool IsControllerLeftHand(GameObject controller, bool actual)
+
+
+        public override bool IsControllerRightHand(GameObject controller)
+        {
+            //bother checking or just return true?
```

Comment:
In your regard if it's always the right controller then yeah can just return true.